dashboards/kubernetes/kubernetes-autoscaling-vpa.libsonnet

```diff
@@ -233,25 +243,27 @@ local tbOverride = tbStandardOptions.override;
         tbQueryOptions.transformation.withOptions(
           {
             renameByName: {
-              verticalpodautoscaler: 'Vertical Pod Autoscaler',
-              container: 'Container',
-              resource: 'Resource',
-              'Value #A': 'Requests',
-              'Value #B': 'Limits',
-              'Value #C': 'Lower Bound',
-              'Value #D': 'Target',
-              'Value #E': 'Upper Bound',
-            },
+                            verticalpodautoscaler: 'Vertical Pod Autoscaler',
+                            container: 'Container',
+                            resource: 'Resource',
+                            'Value #A': 'Requests',
+                            'Value #B': 'Limits',
+                            'Value #C': 'Lower Bound',
+                            'Value #D': 'Target',
+                            'Value #E': 'Upper Bound',
+                          } +
+                          if $._config.vpa.clusterAggregation then { cluster: 'Cluster' } else {},
```

Comment:
Can we leave all of this there by default?

Reply:
That would mean you would always have a column cluster. I am ok with it, here I was trying to get a no-op if this is not enabled.